src/plugins/data-card/components/sort-area.tsx

```diff
@@ -31,16 +34,32 @@ export const DataCardSortArea: React.FC<IProps> = ({ model }) => {
     return placeholders;
   };
 
+  useDndMonitor({
+    onDragStart: (event) => {
+      setDraggingActive(true);
```

Comment:
Since there's a single `DndContext` for the entire document, this will be called for any drag in any component. You probably want to check the id of the draggable to determine whether this particular drag is appropriate to the data card.